src/components/accordion/_accordion.scss

```diff
@@ -140,15 +141,31 @@
       margin-top: -8px;
     }
 
-    .govuk-accordion .govuk-accordion--icon-vertical-bar,
-    .govuk-accordion .govuk-accordion--icon-horizontal-bar {
-      fill: govuk-colour("black");
+    .govuk-accordion__icon::after,
+    .govuk-accordion__icon::before {
+      content: "";
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      width: 25%;
+      height: 25%;
+      margin: auto;
+      background-color: govuk-colour("black");
```

Comment:
could this be `currentColor`? That'll mean that when users change their colours it will override as expected.

Reply:
Thanks @nickcolley. `currentColor` is not supported by IE8. I've just added in a [fix](m/alphagov/govuk-frontend/pull/958/commits/2d7a663b01cc160aa6d1111d26837640b9f77b64) which makes sure that the icon displays for users who change their colours.